Cargo.toml

```diff
@@ -24,3 +24,4 @@ structopt-derive = "0.3"
 url = "2"
 actix-web = "2"
 actix-rt = "1"
+csv = "1"
```

Comment:
do we really need a dependency on csv? Can’t gtfs-structure use its own error, or re-export it?

Reply:
oups you're right I don't need it anymore